src/megaapi_impl.cpp

```diff
@@ -31991,11 +31991,21 @@ bool MegaPushNotificationSettingsPrivate::isGlobalDndEnabled() const
     return (mGlobalDND == 0 || mGlobalDND > m_time(NULL));
 }
 
+bool MegaPushNotificationSettingsPrivate::isGlobalChatsDndEnabled() const
```

Comment:
Isn't this new method the same than `!isChatsEnabled()`?? 
I agree the name `isGlobalChatsDndEnabled()` is more homogeneous, but then... do we need to keep the `isChatsEnabled()`?? multiple interfaces to do the same thing with different code is something to avoid, usually.